package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "generator-cf-component",
+  "version": "0.0.0",
+  "description": "Yeoman generator for Capital Framework components",
+  "license": "MIT",
```

Comment:
According to the [npm docs](https://docs.npmjs.com/files/package.json), the correct way of doing this is now to follow the SPDX license format. In this case it would be: 

```
{ "license" : "SEE LICENSE IN TERMS.md" }
```